processor/src/main/kotlin/com/tobrun/datacompat/DataCompatProcessor.kt

```diff
@@ -187,18 +188,40 @@ class DataCompatProcessor(
                         .returns(Int::class)
                         .build()
                 )
+
+                // Function toBuilder
+                addFunction(
+                    FunSpec.builder("toBuilder")
+                        .addKdoc(
+                            """
+                            Convert to Builder allowing to change class properties.
+                            """.trimIndent()
+                        )
+                        .addStatement(
+                            propertyMap.keys.joinToString(
+                                prefix = "return Builder() .",
+                                transform = { "$it($it)" },
+                                separator = " .",
+                            )
+                        )
+                        .returns(ClassName(packageName, "Builder"))
+                        .build()
+                )
             }
 
             // Builder pattern
             val builderBuilder = TypeSpec.classBuilder("Builder")
             for (property in propertyMap) {
                 val propertyName = property.key.toString()
+                val defaultValue = property.key.annotations
```

Comment:
just checked with empty string `@Default("")` and it doesn't work. maybe add check empty string? you can throw an exception

Reply:
as we're already using String for everything I see no sense in checking one case - even if use "" as default value - it will transform to `val property: String =` and the project will not compile which is OK imo

Reply:
well, as for me, fail fast is better. you can generate code but not immediately compile it.
but up to you